frontend/site/.vitepress/theme/index.js

```diff
@@ -0,0 +1,19 @@
+// https://vitepress.dev/guide/custom-theme
+import Layout from './Layout.vue';
+import "@gouvfr/dsfr/dist/dsfr.min.css";
+import matomo from "@conciergerie.dev/vitepress-plugin-matomo";
+
+/** @type {import("vitepress").Theme} */
+export default {
+  Layout,
+  async enhanceApp({ app, router, siteData }) {
+    if (!import.meta.env.SSR) {
+      await import('@gouvfr/dsfr/dist/dsfr.module');
+    }
+    matomo({
```

Comment:
You can see a call to stats.data.gouv.fr if you build the project with `npm install` from the root project and host it locally instead of using vite dev server.

Matomo tracking is disabled when node environment isn't set to production.